src/Lykke.Job.TransactionHandler/Modules/JobModule.cs

```diff
@@ -270,12 +274,8 @@ private void BindRepositories(ContainerBuilder builder)
                       _dbSettingsManager.ConnectionString(x => x.BitCoinQueueConnectionString), "EthererumPendingActions", _log)));
 
             builder.RegisterInstance<IFeeLogRepository>(
-                new FeeLogRepository(AzureTableStorage<FeeLogEntity>.Create(
-                    _dbSettingsManager.ConnectionString(x => x.FeeLogsConnString), "OrdersFeeLog", _log)));
-
-            builder.RegisterInstance<ITransferLogRepository>(
-                new TransferLogRepository(
-                    AzureTableStorage<TransferLogEntity>.Create(_dbSettingsManager.ConnectionString(x => x.LogsConnString), "TransfersFeeLog", _log)));
+                new FeeLogRepository(AzureTableStorage<FeeLogEntryEntity>.Create(
+                    _dbSettingsManager.ConnectionString(x => x.FeeLogsConnString), "OperationsFeeLog", _log)));
 
```

Comment:
Only the instance of AzureTableStorage should be registered with RegisterInstance method. FeeLogRepository can be registered with RegisterType<>